databricks_cli/utils.py

```diff
@@ -46,9 +46,39 @@ def decorator(*args, **kwargs):
                                'reconfigure with ``dbfs configure``')
             else:
                 error_and_quit(exception.response.content)
-        except Exception as exception: # noqa
+        except Exception as exception:  # noqa
             if not DEBUG_MODE:
                 error_and_quit('{}: {}'.format(type(exception).__name__, str(exception)))
+
+    decorator.__doc__ = function.__doc__
+    return decorator
+
+
+def pipelines_exception_eater(function):
+    """
+    Formats error messages from the pipelines API while keeping the existing
+    behavior of eat_exception
+    """
+
+    @six.wraps(function)
+    def decorator(*args, **kwargs):
+        try:
+            return function(*args, **kwargs)
+        except HTTPError as exception:  # noqa
+            if exception.response.status_code == 401:
+                error_and_quit('Your authentication information may be incorrect. Please '
+                               + 'reconfigure with ``dbfs configure``')
```

Comment:
databricks configure

Reply:
this part is replicating `eat_exceptions`. if we want to change this, it should be in a separate PR and we should do it in both places